infra/terraform/modules/workflow_files/lint.yaml.tftpl

```diff
@@ -43,6 +43,11 @@ jobs:
             REGISTRY=$(grep "REGISTRY_URL := " $MAKEFILE | cut -d\  -f3)
             echo dev-tools=$${REGISTRY}/$${IMAGE}:$${VERSION} >> "$GITHUB_OUTPUT"
           fi
+      - run: docker run --rm %{if lint_env != null }%{ for key, value in lint_env ~}-e ${key} %{ endfor ~}%{ endif }-v $${{ github.workspace }}:/workspace $${{ steps.variables.outputs.dev-tools }} module-swapper%{if lint_env != null }
```

Comment:
@bharathkkb  Shouldn't cause a problem to include, I don't think any of the lint specific ENVs would be used by module-swapper?

Reply:
My thinking was incase we wanted to add any flags for controlling swapper behavior